src/main/js/helpers/constants.js

```diff
@@ -168,6 +168,12 @@ export default {
         top: -10,
         bottom: 20
     },
+    DEFAULT_TICKSCOUNT: 7,
+    TICKSCOUNT_MAXLIMIT: 19,
+    AXISRANGE_1: 20,
+    AXISRANGE_2: 50,
+    AXISRANGE_3: 100,
+    AXISRANGE_4: 500,
```

Comment:
i guess it would be better to rename variables without numbers. Please correct me if i am wrong.

Reply:
If you have alternative suggestions, I'm open to them.

Reply:
you can give something like AXISRANGE_TWENTY or AXISRANGE_ONE orsomething like that, my concern is just to get rid of those numbers.

Reply:
[Fixed](https://github.com/cerner/carbon-graphs/pull/216/commits/d3818be6e4b65ca78df26f01f9cf674fbe9e3398). I'm avoiding having the value in the constant name so that the value can be easily updated without needing to rename the constant.